### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by DMLRBLX** [mulaRahul/keyviz#87] [Ideas]
Hello, first I'd like to say that this software is absolutely the best of its kind! I love the animations and the customization possibilities! With that, there are a few suggestions I'd like to make. The first thing I'd suggest is adding smaller sizes or giving the user the ability to type a size in, instead of having predefined sizes. The second suggestion I have is adding more screen positions instead of just the bottom-left, bottom-right, and bottom-center. I would give the display free-space movement instead of pre-defined options. Those were my suggestions. It can't get any better than that to me though. Keep up the great work!🤘

**Comment by mulaRahul** [comment]
I've already taken the above into consideration as suggested in #14. And I also believe I'll start active development very soon.

**Comment by mulaRahul** [comment]
Added with [v2.0.0a](https://github.com/mulaRahul/keyviz/releases/tag/v2.0.0a).
